tools/myst/enc/enc.c

```diff
@@ -321,6 +322,61 @@ static int _test_oe_debug_mode(void)
     return ret;
 }
 
+static int _get_sgx_config_id(uint8_t out_config_id[64])
+{
+    oe_result_t res = OE_OK;
+    uint8_t* evidence_buffer = NULL;
+    const uint8_t* config_id = NULL;
+    size_t evidence_buffer_size;
+    oe_report_t report;
+    const sgx_report_body_t* sgx_report_body = NULL;
+    /*
+     * Current test cases are written for non ice-lake platform.
+     * Additional code need to be added here to verify the
+     * config id/ svn properties on ice lake platforms
+     */
+    // Generate evidence based on the format selected by the attester.
+    res = oe_get_report_v2(
```

Comment:
Yen suggested to use oe_get_evidence to get the config_id. `oe_get_report_v2` is a legacy API.

https://github.com/openenclave/openenclave/pull/3985/files#r623374878